deploy/operator.yaml

```diff
@@ -19,14 +19,14 @@ spec:
           - /usr/local/bin/ao-logs
           - /tmp/ansible-operator/runner
           - stdout
-          image: image-registry.openshift-image-registry.svc:5000/service-telemetry/service-telemetry-operator:latest
+          image: quay.io/infrawatch/service-telemetry-operator:latest
```

Comment:
Changing the `image` value here is what will be placed into the `CSV` when the bundle is generated.